### PR TITLE
Added English dictionary localization

### PR DESCRIPTION
**Pull request by saketkumar95**
@RocketChat/ReactNative

### CODE REVIEW

File: app/i18n/translations.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import LocalizedStrings from 'react-native-localization';
+
+let strings = new LocalizedStrings({
+ "en-US":{
```
Comment (saketkumar95) [inline app/i18n/translations.js]: @ggazzo We need to add the english text here. I saw that many of them are being updated at run time.

File: android/app/build.gradle [hunk 2]
```diff
@@ -97,7 +97,7 @@ android {
         applicationId "chat.rocket.reactnative"
         minSdkVersion 16
         targetSdkVersion 22
-        versionCode VERSIONCODE as Integer
+        versionCode 57
```
Comment (diegolmello) [inline android/app/build.gradle]: Why did you change this line?

**Comment by mnlbox** [comment]
@diegolmello when you want to merge this PR?

**Comment by ggazzo** [comment]
@mnlbox we are not shure about this library... We really want allow multiples languages soon as possible

**Comment by diegolmello** [comment]
Started from scratch at #312
